metadata-ingestion/src/datahub/ingestion/source/redshift/lineage_v2.py

```diff
@@ -400,13 +400,20 @@ def _process_external_tables(
         db_schemas: Dict[str, Dict[str, RedshiftSchema]],
     ) -> None:
         for schema_name, tables in all_tables[self.database].items():
+            logger.info(f"External table lineage: checking schema {schema_name}")
+            if not db_schemas[self.database].get(schema_name):
+                logger.warning(f"Schema {schema_name} not found")
```

Comment:
We can add some reporting about this. Keeping this info in case this is an unknown yet valid case.
```suggestion
                self.report.info(
                    title="Missing details for schema",
                    message="Lineage will be missing if this is external schema. Check permissions to fix this.",
                    context=schema_name
                )
```